packages/module/patternfly-docs/content/extensions/component-groups/examples/ColumnManagementModal/ColumnManagementModalExample.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { Button } from '@patternfly/react-core';
+import { Table, Tbody, Td, Th, Tr, Thead } from '@patternfly/react-table';
+import ColumnManagementModal, { ColumnManagementModalColumn } from '@patternfly/react-component-groups/dist/dynamic/ColumnManagementModal';
+
+const DEFAULT_COLUMNS: ColumnManagementModalColumn[] = [
+  {
+    title: 'CVE ID',
+    key: 'synopsis',
+    isShownByDefault: true,
+    isShown: true,
+    isAlwaysShown: true
+  },
+  {
+    title: 'Publish date',
+    key: 'publish_date',
+    isShownByDefault: true,
+    isShown: true
+  },
+  {
+    title: 'Impact',
+    key: 'impact',
+    isShownByDefault: true,
+    isShown: true
+  },
+  {
+    title: 'CVSS base score',
+    key: 'cvss_score',
+    isShownByDefault: false,
+    isShown: false
+  }
+];
+
+/* eslint-disable camelcase */
```

Comment:
```suggestion
```
Can we rename the props to camelcase to avoid unnecessary code in the examples. I'm also thinking if it's not better to simplify a bit the names of the cells to be more straightforward (ID, publish date, impact, score) - just to minimize the first-sight complexity of the example. But that's nitpicking